lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -78,10 +94,23 @@ def collect(self, module=None, collected_facts=None):
             if os.path.exists(pkg['path']):
                 pkg_mgr_name = pkg['name']
 
-        if pkg_mgr_name == 'apt' and \
-                os.path.exists('/usr/bin/rpm') and \
-                not os.path.exists('/usr/bin/dpkg'):
-            pkg_mgr_name = 'apt_rpm'
+        # apt is easily installable and supported by distros other than those
+        # that are debian based, this handles some of those scenarios as they
+        # are reported/requested
+        if pkg_mgr_name == 'apt':
+            import q; q(collected_facts['ansible_distribution'])
```

Comment:
shouldn't we look at 'os_family' and any RH based and assume yum/dnf instead?

Reply:
I was going off of `apt` purely because of the precedence model, but I could pivot and focus on distro families.